proposals/0300-Voice-Assistant-App-Service.md

```diff
@@ -93,27 +89,36 @@ We should provide app services for common app types that can integrate with the
 
-3. Once the app is no longer in an HMI level that allows for the service to be active, either the default or previously active voice assistant service with their `onlyActiveWhenVisible` set to `false` will be be reactivated.
 
+1. When the user performs a **short** press on the PTT button, the HMI should trigger core to send a `` RPC to the active Voice Assistant service.
+2. When the user performs a **long** press on the PTT button, the HMI should trigger core to send a `` RPC to the app that is in the foreground.
+ 
 ## Potential downsides
+
 1. Adding an ability to be a voice assistant service for apps like navigation complicates the overall flow of the service. However, by limiting this to only `HMI_FULL` status, we can create an expected flow that must be followed.
+2. HMI's will have to follow the flow provided in this proposal for the PTT button in order to make certain use cases viable. If the OEM chooses not to follow these guidelines, those use cases will not be supported.
 
 ## Impact on existing code
+
 This would have impact on the RPC spec, which impacts Core and all app libraries. Additionally, changes will need to be made to the Policy Server and Developer Portal for this new app service to add a new request from developers to use this app service.
 
 ## Alternatives considered
+
 1. In the original proposal, a voice assistant app service was much more involved. It would actually be given VR synonyms provided by other SDL apps to build out VR grammars and actions. This version keeps things much simpler and allows more apps to be voice assistants. 
 2. Classifying this service as a Voice Recognition service with a capability of being an assistant in the manifest. That way it becomes a little less cloudy of what the service does. The change would be easy to make if the SDLC decides that is a better option to go.
+3. Adding the ability to have temporary active services for this service type. This would then allow apps that just want to use VR while in the foreground to use services. This introduced a lot of complexity that was not necessary in the end.
+4. During the revisions period, it was found that the `OnVoiceSessionStarted` notification should be changed to a request/response pair instead, named `InitiateVoiceSession`. This will allow any app that receives the request to respond with unhandled voice data. This will be important for the navigation use case where it is the app to start the voice session. this would also remove the param for the `VoiceAssistantServiceData` struct.
```

Comment:
```suggestion
4. During the revisions period, it was found that the `OnVoiceSessionStarted` notification should be changed to a request/response pair instead, named `InitiateVoiceSession`. This will allow any app that receives the request to respond with unhandled voice data. This will be important for the navigation use case where it is up to the app to start the voice session. This would also remove the param for the `VoiceAssistantServiceData` struct.
```